packages/flutter_tools/lib/src/base/utils.dart

```diff
@@ -8,6 +8,12 @@ import 'dart:io';
 import 'package:crypto/crypto.dart';
 import 'package:path/path.dart' as path;
 
+bool isRunningOnTravis() {
+  // https://docs.travis-ci.com/user/environment-variables/#Default-Environment-Variables
+  return Platform.environment['TRAVIS'] == 'true' ||
+    Platform.environment['CONTINUOUS_INTEGRATION'] == 'true';
```

Comment:
for clarity we usually line up the matching parts on things like this (so the Ps would be lined up)